postgres/tests/unit.tftest.hcl

```diff
@@ -71,6 +71,292 @@ run "aws_security_group_unit_test" {
   }
 }
 
+# Uncomment as part of https://uktrade.atlassian.net/browse/DBTP-1117
+
```

Comment:
would it not be worth getting rid of all of the commented lines below to keep this ticket clean and just add them when #DBTP-1117 is taken care of? other than that, lgtm 👍🏻

Reply:
You're right. I've added the work as comments on this other ticket, in case they're useful later https://uktrade.atlassian.net/browse/DBTP-1117